packages/cli/src/cmds/assert/strategy/httpRequestStrategy.ts

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore
+import { Event } from '@appland/models';
+import Strategy from './strategy';
+import { Scope } from '../types';
+
+export default class HttpRequestStrategy extends Strategy {
+  protected scope: Scope = 'http_server_request';
+
+  protected isEventApplicable(event: Event): Boolean {
+    return event.httpServerRequest && event.elapsed;
```

Comment:
Here is `event.elapsed` again, which I think will always be null.

Reply:
Is it enough to check for `event.httpServerRequest`?

Reply:
Yes. `elapsed` is actually on the return event anyway.